src/test/features/claim/routes/pay-by-account.ts

```diff
@@ -75,7 +75,11 @@ describe('Claim : Pay by Fee Account page', () => {
 
     it('should return 500 and render error page when form is valid and cannot save draft', async () => {
       idamServiceMock.resolveRetrieveUserFor('1', ...roles)
+      idamServiceMock.resolveRetrieveServiceToken()
       draftStoreServiceMock.rejectSave(100, 'HTTP error')
+      feesServiceMock.resolveCalculateIssueFee()
```

Comment:
I guess case was failing with 500 before which was caused by missing mocks.

Reply:
this file had soo many connection refused happening and not doing what the tests description was.
I fixed that